features/security/src/main/feature/feature.xml

```diff
@@ -175,28 +175,31 @@
         <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.opensaml/3.1.1_3
         </bundle>
 
-        <bundle>mvn:org.codice.thirdparty/cas-client-core/${cas.client.bundle.version}</bundle>
+        <bundle>wrap:mvn:org.jasig.cas.client/cas-client-core/${cas.client.version}</bundle>
```

Comment:
❓ Is there a reason we're not updating the version of `mvn:org.codice.thirdparty/cas-client-core` in the thirdparty repo instead of wrapping it?

Reply:
I did it this way because using karaf's wrapping capability seemed simpler and I've seen it used elsewhere. Is wrapping it ourselves preffered?

Reply:
I don't think we have a preferred way to do this. I was mostly commenting to point out we're doing both methods to accomplish the same thing. Wrapping is fine :+1: